README.md

```diff
@@ -85,13 +70,16 @@ npm run build-test
 
 ### Integration Tests
 Each package with integration tests has their own instructions for running:
+- [Core](packages/amazon-sumerian-hosts-core/test/integration_test/README.md)
 - [Babylon.js](packages/amazon-sumerian-hosts-babylon/test/integration_test/README.md)
 - [Three.js](packages/amazon-sumerian-hosts-three/test/integration_test/README.md)
 
 ### Examples Folder
 Each of the implementation packages have their own examples attached to them. Each of these files contain enough code to stand up and verify the code is working properly. To verify, open up a local server for the package you are modifying:
 ```
-npm run --workspaces=./packages/amazon-sumerian-hosts-{package} start
+npm run start-{engine-name} start

```

Comment:
i think the last `start` is not necessary

Reply:
yep, thanks!